.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+# Issue Title
+
+Be as descriptive as you can with your title. If your issue requests
+a new topic, preface the title with [NewTopic].
+
+# General
+
+We use issues to drive the discussion for changes to
+existing topics and the creation of new topics.
+
+This way, we have those discussions before significant
+effort is spent writing a new topic. Click on the Guidelines
+for Contributing link above for details.
+
+## Issues with Existing Topics
+
+If the issue is a simple typo or similar correction, just submit a PR.
```

Comment:
Instead of "just submit a PR", maybe we should say you can just submit a PR. They can still open an issue if they prefer IMO,